less/dropdowns.less

```diff
@@ -10,7 +10,7 @@
   height: 0;
   margin-left: 2px;
   vertical-align: middle;
-  border-top:   @caret-width-base dashed;
+  border-top:   @caret-width-base solid;
```

Comment:
Fixed redundant line(s) (one on this and one at 189) and made the change to line 189. Sorry about missing those